src/aws-api-mcp-server/tests/common/test_helpers.py

```diff
@@ -106,6 +108,20 @@ def test_get_requests_session():
     assert retry_config.allowed_methods == {'HEAD', 'GET', 'OPTIONS', 'POST'}
 
 
+def test_json_serializer_datetime():
```

Comment:
This feels like a low coverage for the change. It effectively tests built in functions, isoformat() and str(obj). Instead of testing the actual change. I'd expect more e2e tests that actually verify support for new commands